docs/_config_local.yml

```diff
@@ -0,0 +1,25 @@
+theme: "just-the-docs"
+
+exclude: ["node_modules/", "*.gemspec", "*.gem", "Gemfile", "README.md","Gemfile.lock", "package.json", "package-lock.json",  "script/", "LICENSE.txt", "lib/", "bin/", "README.md", "Rakefile"]
+
+
+aux_links:
+  "VulnerableApp":
+    - "https://github.com/SasanLabs/VulnerableApp"
+
+title: Vulnerable App
+description: Documentation for VulnerableApp
+baseurl: "/sasn" # the subpath of your site, e.g. /blog
```

Comment:
what does this do ?

Reply:
this tells the templating engine that builds the site the baseurl that needs to show in the browser